### PR TITLE
Lots more examples and exercises

### PR DESCRIPTION
**Pull request by toppsdown**
Still working on this but wanted to get feedback @nw0428

### CODE REVIEW

File: app/controllers/completed_exercises_controller.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
+class CompletedExercisesController < ApplicationController
```
Comment (nw0428) [inline app/controllers/completed_exercises_controller.rb:1]: I think if we can get through JUST this controller and its routes (making a route and then its method each time) than we have made a gigantic improvement to the course 👍

File: app/controllers/example_controller.rb [hunk 2]
```diff
@@ -12,6 +12,18 @@ def example3
   end
 
   def example4
```
Comment (nw0428) [inline app/controllers/example_controller.rb]: Feel free to grab just fizzbuzz out of here and move it into either of the other controllers

**Review by nw0428 — APPROVED** [review]
LRGTM 👍

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,138 @@
...
+
+  # All of these examples do the same thing, but
+  # using different HTTP verbs
+
+  # Only GET works from a browser URL
   get 'example1', to: "example#example1"
-  post 'example2', to: "example#example2"
-  put 'example3', to: "example#example3"
-  delete 'example4', to: 'example#example4'
+  put 'example1', to: "example#example1"
```
Comment (toppsdown) [inline config/routes.rb]: @nw0428 Can you add the curl commands as comments to this so the students have a reference?

**Comment by toppsdown** [comment]
@nw0428 @chdastolfo I think this is all set, gonna merge this.

**Pull request merged**